modular_skyrat/modules/borg_buffs/code/robot.dm

```diff
@@ -116,3 +116,78 @@
 		tool_behaviour = TOOL_KNIFE
 		to_chat(user, span_notice("You attach the knife bit to [src]."))
 		icon_state = "knife_screw_cyborg"
+
+/obj/item/inducer/cyborg
+	name = "Cyborg Inducer"
+	desc = "A tool for inductively charging internal power cells using the battery of a cyborg"
+	powertransfer = 250
+
+
+
+/obj/item/inducer/cyborg/attackby(obj/item/W, mob/user)
+	if(W.tool_behaviour == TOOL_SCREWDRIVER)
+		return
+	if(istype(W, /obj/item/stock_parts/cell))
+		return
+
+/obj/item/inducer/cyborg/recharge(atom/movable/A, mob/user)
+	if(!iscyborg(user)) //
```

Comment:
A comment born prematurely, perhaps?

Reply:
minor leftover. i missed it while cleaning up